packages/adminrouter/extra/src/.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+
+# Ignore the following errors:
```

Comment:
Nit: I share your opinion on style here, but we should probably try to make this identical to the style used on the rest of the python in this repo if its living here forever